source/style/view.css

```diff
@@ -1,79 +1,55 @@
 @import "constants.css";
-@import "global.css";
-
-/* Margin between two buttons should be 60px, but should double check with NEW constants.css page
-    or check with other teams? */
-.right-button-container {
-    right: 0;
-    position: relative;
-    float: right;
-}
+@import "global.css"; 
 
+/* set background color */
 body {
     background-color: var(--secondary-color);
 }
 
-.outerframe {
-    position: relative;
-    width: 900px;
-    height: 886px;
-    left: relative;
-    top: 134px;
-    overflow-y: scroll;
-    margin: auto;
-    
-}
-
+/* define dish-name style*/
 .dish-name {
-    position: center;
     margin-top: 41px;
     text-align: center;
-    font-size: 48px;
     line-height: 59px;
 }
 
-.ingredients {
-    padding-top: 50px;
-
+/*define dish-name style*/
+.dish-name h1 {
+    font-size: var(--h1-font-size);
+    font-weight: 800px;
+    font-weight:600;
 }
 
-.steps {
-    padding-top: 40px;
-}
+/*define padding for each block*/
+.block{
```

Comment:
if we change `<div class="block">` to `<section>` tags, this would need to be renamed too